pipeline/test_extract.py

```diff
@@ -0,0 +1,79 @@
+# pylint: skip-file
+"""Tests for the test_extract.py script."""
+
+import pytest
```

Comment:
```suggestion
from pytest import raises
```

I'll make the suggestions for the bits you need